src/main/resources/db/migration/V2020_02_07_00_27__Add_Material_Info_To_Required_Material.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE `4ncov`.`material_required`
```

Comment:
No need to specify schema here since already defined when importing flyway. This will lead the sql migration to schema `4ncov` instead of the expected schema.